lib/pbench/server/indexer.py

```diff
@@ -62,10 +62,7 @@
 #       - code structure is changed
 #       - additional indexes are added without changing existing indices or
 #         mappings
-#     patch: change when:
-#       - bug fixes are made to existing code that don't affect indices or
-#         mappings
-# We add the version to the run documents we generate.  Having the version
+#     patch: change when:datasetsun documents we generate.  Having the version
```

Comment:
Oops; I've really been having trouble avoiding touching my laptop trackpad while typing on this ridiculously high "desk" at Quality Inn; it keeps selecting ranges and deleting things on me. I thought I'd found them all, but obviously not...

Reply:
Set off a time bomb to remove the variant ...